src/benchmark/operators/insert_constraint_benchmark.cpp

```diff
@@ -0,0 +1,117 @@
+#include <memory>
+#include <vector>
+
+#include "../micro_benchmark_basic_fixture.hpp"
+#include "benchmark/benchmark.h"
+#include "concurrency/transaction_manager.hpp"
+#include "operators/get_table.hpp"
+#include "operators/insert.hpp"
+#include "operators/table_wrapper.hpp"
+#include "storage/storage_manager.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+auto reset_table(bool use_constraints, int values_to_insert, int num_rows = 0) {
```

Comment:
We don't use `auto` for return types (except for lambdas and where unavoidable). I shouldn't have to read the method to understand its signature.